theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
@lamblin In fact, GpuDimShuffle inherits from DimShuffle which is now a COp. But DimShuffle has an `__init__()` in which there is a call to `COp.__init__(self, [filenames], function_name)`. Then this `__init__()` is inherited into GpuDimShuffle, even if C functions are hidden.

I don't know how to bypass that. Maybe should I just rewrite GpuDimShuffle too in that PR as soon as possible ? (Monday, I think !)